functions/form-payment-reconciliation/index.js

```diff
@@ -0,0 +1,149 @@
+const { SSMClient, GetParameterCommand } = require('@aws-sdk/client-ssm')
+const { generateLinkToCloudwatch, postToSlack } = require('./src/utils')
+
+const AWS_REGION = 'ap-southeast-1'
+
+const PARAMETER_STORE_NAME = 'staging-cron-payment'
+
+const SUB_DOMAIN = process.env.ENV_SITE_SUBDOMAIN || 'staging.'
+
+/** TODO: migrate to separate file */
+function getSecrets() {
+  const awsSsmClient = new SSMClient({ region: AWS_REGION })
+  const input = {
+    Name: PARAMETER_STORE_NAME,
+    WithDecryption: true,
+  }
+  const command = new GetParameterCommand(input)
+  return awsSsmClient
+    .send(command)
+    .then((d) => d.Parameter.Value)
+    .then((d) => {
```

Comment:
Why 2 promise steps? Is `d.Parameter.Value` a promise?

Reply:
also fixed in [95ab1e8](https://github.com/opengovsg/FormSG/pull/6102/commits/95ab1e89b5031f60b29d3019a75651cfb24f589d)